_docs_integrate/error-codes.md

```diff
@@ -35,6 +35,8 @@ Please find a list of enmeshed error codes below. Most often the errors occur on
 | {% include anchor a="error.connector.attributes.cannotCreateNotSelfOwnedRepositoryAttribute" %}             | When creating a RepositoryAttribute the owner must not be someone else.                                                                                                                                                                                                   |
 | {% include anchor a="error.consumption.attributes.cannotSucceedAttributesWithAParent" %}                    | The Attribute you want to succeed has a parent. You cannot succeed Attributes with a parent. Instead, succeed the parent, which will implicitly succeed all its children. For example, if you want to change a HouseNumber, you should succeed the Address it is part of. |
 | {% include anchor a="error.consumption.requests.attributeQueryMismatch" %}                                  | The provided Attribute does not match the AttributeQuery.                                                                                                                                                                                                                 |
+| {% include anchor a="error.consumption.requests.wrongRelationshipStatus" %}                                 | The Relationship needs to be in a different status (e.g. Active) to send/decide that Request.                                                                                                                                                                             |
+| {% include anchor a="error.consumption.requests.missingRelationship" %}                                     | You need a Relationship to send/decide that Request.                                                                                                                                                                                                                      |
```

Comment:
this table makes no sense if you just copy paste the messages from the libraries here (as you will get exactly this message when sth. goes wrong). Especially this message / description is really saying nothing without context. Try to enhance it.